csharp/GildedRose.cs

```diff
@@ -1,88 +1,72 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization.Formatters;
 
 namespace csharp
 {
     public class GildedRose
     {
-        IList<Item> Items;
-        public GildedRose(IList<Item> Items)
+        IList<Item> items;
+        public GildedRose(IList<Item> items)
         {
-            this.Items = Items;
+            this.items = items;
         }
 
         public void UpdateQuality()
         {
-            for (var i = 0; i < Items.Count; i++)
+            for (var i = 0; i < items.Count; i++)
             {
-                if (Items[i].Name != "Aged Brie" && Items[i].Name != "Backstage passes to a TAFKAL80ETC concert")
+                if (!items[i].Name.Contains("Sulfuras"))
```

Comment:
I would probably create a class for each item type, starting from an abstract class that extends item, having a method `UpdateQuality`. Then, you could just call `item.UpdateQuality()`.

The advantage is that you can separate the logic for updating the quality of each item, and you can easily add new item types without changing this logic